rust_snuba/src/strategies/clickhouse/batch.rs

```diff
@@ -70,6 +71,12 @@ impl BatchFactory {
             }
         }
 
+        if max_bytes_before_external_group_by.is_some() {
+            let mut query_segment: String = "&max_bytes_before_external_group_by=".to_owned();
+            query_segment.push_str(&max_bytes_before_external_group_by.unwrap().to_string());
+            query_params.push_str(&query_segment)
+        }
```

Comment:
Nit: I recommend the following to avoid the `.unwrap` call
``` rust
if let Some(max_bytes_before_external_group_by) = max_bytes_before_external_group_by {
    query_segment.push_str(max_bytes_before_external_group_by);
    ...
}
```
Also, follow up question:
how come we are pushing the value in before the option name? Should the order be opposite where we push the option name first, then the value?

Reply:
Taking your first suggestion into account.

For the second, I think the way push_str works this should work out to "&max_bytes_before_external_group_by=500000" (or whatever value in place of 500k), right?

Reply:
Ah yes you're right, I misread the code.